samples/open-telemetry/otel.yaml

```diff
@@ -1,58 +1,71 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: otel-collector-conf
-  namespace: istio-system
+  name: opentelemetry-collector-conf
```

Comment:
should we keep the namespace? all the other samples have it baked in?

Reply:
that would forbidden user install the other namespace?
```console
kubectl apply -f samples/open-telemetry/otel.yaml  --namespace=default
the namespace from the provided object "istio-system" does not match the namespace "default". You must pass '--namespace=istio-system' to perform this operation.
```

Reply:
It is just a sample though, they can change it. Also the meshconfig in docs, etc will use the hardcoded namespace

Reply:
upadted

Reply:
ping @howardjohn